app/fullService/api/submitTransaction.ts

```diff
@@ -30,10 +30,11 @@ const submitTransaction = async ({
 
   if (error) {
     throw new Error(error);
-  } else if (!result) {
-    throw new Error('Failure to submit transaction.');
-  } else {
+    // submitting a tx created in offline mode will not return a transaction log
```

Comment:
fixes a bug where an error would render even though the tx was submitted sucessfuly